language/use-cases/sql-code-generation/sql_code_generation.ipynb

```diff
@@ -0,0 +1,1609 @@
+{
```

Comment:
<p>Since there's new models available, maybe the model name should be a variable/constant with <code>@param {type:"string"}</code> for easy updates?</p><p> </p><p>Also, is there a reason you're using <code>text-bison</code> instead of <code>code-bison</code> ?</p><p> </p><p><a href="https://cloud.google.com/vertex-ai/docs/generative-ai/code/code-models-overview#:~:text=x-,GoogleSQL,-.sql" target="_blank">Codey should support GoogleSQL</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>